src/app/models.rs

```diff
@@ -75,6 +75,9 @@ pub struct AlbumDescription {
     pub artists: Vec<ArtistRef>,
     pub art: Option<String>,
     pub songs: Vec<SongDescription>,
+    pub label: String,
```

Comment:
It has broken more then the artists page. I have to look at whats wrong. I think it may have to do with artist_album not including label and copyright info

Reply:
I think I fixed for search and artist page, now I discovered that this is a problem too with playlist as those dont list release dates. This is an issue with adding the fields label, release_date, and copyrights to Album

Reply:
ok I see! if that is easier, we could have separate Album API model for endpoints that return a "light" version of it. that might be easier to handle than all the Options you'll have to deal with!

Reply:
Ye probably. Earlier on I had a struct that had all the information that AlbumDescription had + the extra info. Seeing that this info is only relevant for this screen maybe I could make it hold AlbumDetailedDescription that contains the details, and the extra info